src/production/models.py

```diff
@@ -66,3 +66,13 @@ def status(self):
                 self.completed.strftime("%Y-%m-%d %H:%M"))}
         elif self.completed and self.editor_reviewed:
             return {'slug': 'closed', 'friendly': 'Task closed'}
+
+    @property
+    def get_status_code(self):
+        return self.status().get('slug')
```

Comment:
The way in which `TypesetTask.status` is coded it could return `None`, which would throw an `AttributeError` here. I think a cleaner implementation would be for `TypesetTask.status` to return the status code and some other `TypesetTask.slug` which calls `TypesetTask.status` and returns a slug structure.

Reply:
Done.